sim/dex-moves.ts

```diff
@@ -203,7 +203,16 @@ export interface MoveData extends EffectData, MoveEventMethods, HitEffect {
 	basePowerModifier?: number;
 	critModifier?: number;
 	critRatio?: number;
-	defensiveCategory?: 'Physical' | 'Special' | 'Status';
+	/**
+	 * controls which pokemon it draws the attack stats from. This does not use the pokemon's ability or item
+	 */
+	overrideOffensivePokemon?: 'target' | 'source';
+	overrideOffensiveStat?: StatIDExceptHP;
+	/**
+	 * controls which pokemon it draws the defense stats from. This does not use the pokemon's ability or item
+	 */
+	overrideDefensivePokemon?: 'target' | 'source';
+	overrideDefensiveStat?: StatIDExceptHP;
```

Comment:
```suggestion
	/** Pokemon for the attack stat. Ability and Item damage modifiers still come from the real attacker. */
	overrideOffensivePokemon?: 'target' | 'source';
	overrideOffensiveStat?: StatIDExceptHP;
	/** Pokemon for the defense stat. Ability and Item damage modifiers still come from the real defender. */
	overrideDefensivePokemon?: 'target' | 'source';
	overrideDefensiveStat?: StatIDExceptHP;
```

We still need an explanation for Choice Band. Does it modify the Attack stat?

Reply:
added this